src/components/locationLabel.js

```diff
@@ -17,7 +18,11 @@ function LocationLabel(props) {
   }
   return (
     <LocationLabelContainer>
-      <h1>{props.city}</h1>
+      <div style={{ display: "flex", alignItems: "center" }}>
+        <img src={icon} alt="icon" width="42" heigh="42" className="icon" />
+        <h1>{props.city}</h1>
```

Comment:
Super el icono aqui, solo dos cosas: 

1. Seria super la temperatura actual ahi tambien, no solo el icono
2. Cuando el background esta muy oscuro, no se lee nada. Puedes hacer algo [como esto](https://stackoverflow.com/questions/11867545/change-text-color-based-on-brightness-of-the-covered-background-area) para elegir el color de la letra, o darle un background color al div para que no importa el color de la foto, siempre se lea.

![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZDkwNzU3M2UtNWNkYi00Y2FmLTkwNjItZWIyZTAzZjhhY2NhIiwiZGVwbG95X2lkIjoiNjE2MWQ2ODU3OTEyYWUwMDA4Y2Q1ODYzIiwiaWQiOiI4YjA5M2E4Ny02NzAzLTRlNTQtYjAxMi0wOTVlOGRmN2JmYzgifQ==)